QUICKSTART-DEPLOY.md

```
@@ -10,13 +10,31 @@
 ## Step 1: Start the Platform
 
-1. Start all services using Docker Compose:
+1. Change to the deployment directory:
 
 ```bash
-docker-compose up metald-aio dashboard ctrl -d
+cd deployment
 ```
 
-2. Wait for all services to be healthy
+2. Create an `.env` file in `apps/dashboard` (use .env.example as a template).
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)